views/pages/HomePage.jsx

```diff
@@ -54,70 +58,68 @@ const HomePage = () => {
       setNotification({ open: true, message: error.message })
     }
   }
-  const sort = async (newSortBy, newSortDirection) => {
-    const participantList = await fetchParticipants({
-      sortBy: newSortBy,
-      sortDirection: newSortDirection,
-      searchSubjects: normalizeSearchSubjects(searchSubjects),
-    })
-
+  const onSort = async (newSortBy, newSortDirection) => {
     setSortBy(newSortBy)
     setDirection(newSortDirection)
-    setParticipants(participantList)
   }
 
-  const handleSearch = async (e) => {
-    setSearchSubjects(e)
-
-    const participantsList = await fetchParticipants({
-      sortBy,
-      sortDirection,
-      searchSubjects: normalizeSearchSubjects(e),
-    })
-
-    setParticipants(participantsList)
-    setRowcount(participantsList.length)
-
-    if (e.length === 0) {
-      const participantsList = await fetchParticipants()
-
-      setParticipants(participantsList)
-    }
+  const handleSearch = async (formData) => {
+    setSearchSubjects(formData.participants)
   }
   const normalizeSearchSubjects = (searchSubjects) =>
     searchSubjects.map(({ value }) => value)
 
   useEffect(() => {
     fetchParticipants().then((participantsList) => {
-      const dropDownOptions = participantsList.map(({ study, subject }) => ({
-        value: `${subject}`,
-        label: `${subject} in ${study}`,
-      }))
+      if (initialLoad) {
+        const dropDownOptions = participantsList.map(({ study, subject }) => ({
+          value: `${subject}`,
+          label: `${subject} in ${study}`,
+        }))
+        setSearchOptions(dropDownOptions)
+        setInitialLoad(false)
+      }
 
       setParticipants(participantsList)
-      setSearchOptions(dropDownOptions)
     })
-  }, [])
+  }, [fetchParticipants, setParticipants, setSearchOptions])
 
   return (
     <>
-      <ReactSelect
-        placeholder="Search a study or participant"
-        value={searchSubjects}
-        onChange={handleSearch}
-        options={searchOptions}
-        autoFocus={true}
-        components={components}
-        isMulti
-      />
-      <ParticipantsTable
-        participants={participants}
-        onUpdate={handleUserUpdate}
-        onSort={sort}
-        sortProperty={sortBy}
-        sortDirection={sortDirection}
-        sortable
-      />
+      <Box sx={{ p: '20px' }}>
+        <Box
+          sx={{
+            mb: '20px',
+            display: 'flex',
+            gap: '16px',
+            justifyContent: 'space-between',
+            alignItems: 'center',
+          }}
+        >
+          <Typography sx={{ fontWeight: 600, width: 192 }}>
```

Comment:
should this use a maxWidth..?

Reply:
This is a funky one. I think we'll have a better solution here as we implement more of these pages, but there are consistent designs for the top search forms and the area for the "title" of those forms has a width of 192 pixels.